### PR TITLE
Feature - RA

### PR DESCRIPTION
**Pull request by joefitter**
* Added ra route for authoring/reading
* Updated project version middleware to be used for RA versions
* Added ra type to project tasks
* Updated NTS to include granted RA if present
* Updated Project landing page to include link to submit RA

### CODE REVIEW

**Comment by joefitter** [comment]
WIP until https://github.com/UKHomeOffice/asl-projects/pull/677

**Comment by joefitter** [comment]
🚢

File: pages/retrospective-assessment/index.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+const bodyParser = require('body-parser');
+const { get } = require('lodash');
+const routes = require('./routes');
+const { getComments, getChangedValues } = require('../project-version/middleware');
+const extractComments = require('../project-version/lib/extract-comments');
+
+module.exports = () => {
+  const app = Router({ mergeParams: true });
+
+  app.use(bodyParser.json({ limit: '5mb' }));
```
Comment (lennym) [inline pages/retrospective-assessment/index.js:11]: Not sure if this is copied from somewhere, but can we avoid the hard-coded limit and use the limit from config?

See https://github.com/UKHomeOffice/asl-pages/blob/master/pages/project-version/update/router.js#L76

Comment (joefitter) [inline pages/retrospective-assessment/index.js:11]:
```suggestion
  app.use(bodyParser.json({ limit: settings.bodySizeLimit }));
```

File: pages/retrospective-assessment/index.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+const { Router } = require('express');
+const bodyParser = require('body-parser');
+const { get } = require('lodash');
+const routes = require('./routes');
+const { getComments, getChangedValues } = require('../project-version/middleware');
+const extractComments = require('../project-version/lib/extract-comments');
+
+module.exports = () => {
```
Comment (joefitter) [inline pages/retrospective-assessment/index.js:8]:
```suggestion
module.exports = settings => {
```

**Pull request merged**